ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetTransactionCountTest.java

```diff
@@ -56,4 +75,107 @@ public void verifyTransactionCount() {
 
     assertThat(response.getResult()).isEqualTo(String.format("0x%X", NONCE));
   }
+
+  @Test
+  public void validRequestProducesExpectedNonce() {
```

Comment:
Rename this test to indicate it testing that valid legacy requests have the expected nonce. Can another test be added for the pantheon case?

Reply:
Changed test names and added another test